at-spi2-core: propagate glib and dbus

Profpatsch opened the pull request:
Both libraries are in the pkgconfig `Requires` field and are needed by
all downstream dependencies of this library.

Fell out of https://github.com/NixOS/nixpkgs/pull/78051

Profpatsch committed:
at-spi2-core: propagate glib and dbus

Both libraries are in the pkgconfig `Requires` field and are needed by
all downstream dependencies of this library.

worldofpeace commented on pkgs/development/libraries/at-spi2-core/default.nix:
```diff
@@ -30,7 +30,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ meson ninja pkgconfig gobject-introspection makeWrapper ];
-  buildInputs = [ dbus glib libX11 libXtst libXi ];
+  buildInputs = [ libX11 libXtst libXi ];
+  propagatedBuildInputs = [ dbus glib ];
```
```suggestion
  # in atspi-2.pc dbus-1 glib-2.0
  propagatedBuildInputs = [ dbus glib ];
```

I like to comment that so I can keep an eye on the pc files.
(really would like to figure out why we patch pkgconfig someday)

worldofpeace reviewed (commented):
Should be fine, can't remember how big the rebuild is here (waiting on eval)

worldofpeace commented:
Ok this is huge :exclamation: staging please :smile:

Profpatsch commented:
Grr, of course it requested reviews of everyone on the planet. Fix your shit Github.

How come I rebased on staging (I fetched the branch before rebasing) and it’s not only the commits I added?

Infinisil commented:
I think you need to rebase on the merge-base of master and staging, e.g `git fetch upstream; git rebase --onto $(git merge-base upstream/master upstream/staging) HEAD~1`

Profpatsch commented:
Ah, yes, obviously. Sometimes I forget how to git …

Profpatsch commented:
@worldofpeace we can merge into staging when the evaluation succeeds, right?

worldofpeace reviewed (approved):
Yep @Profpatsch

worldofpeace commented:
> Ah, yes, obviously (`r s` in magit). Sometimes I forget how to git …

Literally mood every day :rofl:
